recipes/google-cloud-cpp-compute/run_test_feature.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Stop on first error
+set -euo pipefail
+
+if [[ "${target_platform}" == osx-* ]]; then
+  # as in build.sh
+  CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+feature=${PKG_NAME/#libgoogle-cloud-/}
+feature=${feature/%-devel/}
```

Comment:
Same here actually - I wouldn't even know how that works. Are `#`/`%` a different spelling of regex(-like) markers?

Reply:
Added a comment.

https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
OK, it _is_ a pattern-matching syntax. It's rare enough that I would have just put the link into the comment, but fine.